torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -2488,6 +2494,17 @@ Tensor* tensorexpr::computeOperandValue(
     }
     case aten::transpose: {
       auto A = c10::get<BufHandle>(inputs[0]);
+      // Trivial case of 0-dim and 1-dim tensors: transpose is just a copy
+      if (A.ndim() < 1) {
+        return Compute(
+            "aten_transpose",
+            c10::fmap<DimArg>(outputShape),
+            [&](std::vector<VarHandle> axes) {
+              TORCH_INTERNAL_ASSERT(axes.size() <= 1);
+              return A.load(axes);
+            });
```

Comment:
On the second thought on this - I feel that such a method won't buy us much since we'd need to pass lots of things into it (A, outputShape, name_hint) and it will only be used in this one place. Would you be ok if we keep the code as is? The other two "trivial" computes are also different - they are accessing the 0th element of a 0-dim input tensor, this one works for 1-d and 0-d.